README.md

```diff
@@ -82,4 +82,4 @@ You can use function by typing "$name" in staff chat and will be replaced with a
 | staffchat.check  | Allow players to check other player's staff chat status|
 | staffchat.list   | Allow players to list players staff chat status        |
 
-By default all permission nodes are granted for OPs
+By default all permission nodes are granted for OPs, this is not meant to work for console!
```

Comment:
this is basically false, i craft this to allow it to work with console and players
see /sc say and /sc attach
therefore console should be able to do the same thing as a player could, other than the "/sc on/off" part